ChangeLog.md

```diff
@@ -1,5 +1,13 @@
 # Revision history for haskell-lsp
 
+## 0.20.0.0
+
+* Don't log errors for '$/' notifications (@jinwoo)
+* Force utf8 encoding when writing vfs temp files (@jneira)
+* Store a hash in a NormalizedUri (@mpickering)
+* Move "Semigroup WorkspaceEdit" instance (@sheaf)
+* Fix vfs line endings (@merged)
```

Comment:
```suggestion
* Fix vfs line endings (@jneira)
```

Reply:
oops. My attention to detail sucks tonight :(